client/src/js/directives.js

```diff
@@ -34,22 +39,24 @@ module.exports = function (app) {
            * @param {string} url relative URL to icon
            * @returns {string} Icon Url to Assets
            */
-          $scope.createIconUrl = function (url) {
+          $scope.createIconUrl = function createIconUrl(url) {
             return ASSET_URL + url;
           };
 
           $scope.toggleSubmenu = [];
-          $scope.showSubmenu = function (index) {
-            for (var i = 0; i < $scope.toggleSubmenu.length; i++) {
-              if (i !== index) {
-                $scope.toggleSubmenu[i] = false;
+          $scope.showSubmenu = function showSubmenu(indexCheck) {
+            let index = 0;
+
+            for (; index < $scope.toggleSubmenu.length; index++) {
```

Comment:
@valorkin Can we relax a bit eslint-rules in order to avoid such kind of cases ?
```
let index = 0;
for (; index < $scope.toggleSubmenu.length; index++) {
```
Because now es-lint want us to hoist all the variable declaration to the top of the scope